webapp/src/modules/wallet/sagas.js

```diff
@@ -107,7 +107,7 @@ function* handleConnectWalletRequest(action = {}) {
       manaTokenContract.balanceOf(address),
       fetchBalance(address),
       manaTokenContract.allowance(address, marketplaceAddress),
-      landRegistryContract.isApprovedForAll(marketplaceAddress, address),
+      landRegistryContract.isApprovedForAll(address, marketplaceAddress),
```

Comment:
⚠️ do we want this on master already?
(I think we need to downgrade the contract and re-deploy the new one on another address)

Reply:
I can add a comment to revert that change before marge it to master. What do you think?